content/influxdb/v2.0/upgrade/v1-to-v2/_index.md

```diff
@@ -136,6 +136,9 @@ Consider [enabling authentication in your InfluxDB 1.x instance](/influxdb/v1.8/
 **before** upgrading to InfluxDB 2.0 to ensure the appropriate credentials are in place and
 that the various applications, agents, and visualization tools are able to connect.
 
+If `auth-enabled = false` is set in your InfluxDB 1.x configuration, after upgrading to InfluxDB 2.0,
+clients sending requests (for example, Telegraf) will have invalid credentials and requests will be ignored (and return `401 Unauthorized`).
```

Comment:
this isn't what happens.

I would suggest the following:
If you’ve been running with `auth-enabled = false` in your 1.x setup, a warning is now produced as this is not an option in 2.x.  If you see this warning, it is recommended to reconfigure your 1.x instance with `auth-enabled = true` and ensure all of the username/password combinations are working as expected before retrying the upgrade process.

In my situation, I THOUGHT that auth was enabled and was being enforced as I had username and passwords everywhere.  However, when I upgraded, it turns out that one of the passwords that I was using was incorrect -- but since this was NOT being enforced, it only caused problems once the upgrade was completed.  So, the process outlined above is solid guidance to ensure that everything is secure and working before upgrading....i.e. work out the security bits on your known good environment first, then proceed with the upgrade.  Think about it like this... turning auth on is a prerequisite to performing the upgrade.

Reply:
Thanks, @timhallinflux.